liquibase-service/src/main/resources/db/rdb/routines/001-generate_data_for_data_sync_config-002.sql

```diff
@@ -0,0 +1,40 @@
+INSERT INTO [RDB].[dbo].[data_sync_config]
+(table_name, source_db, query, query_with_null_timestamp, query_count, query_with_pagination)
+VALUES
+    ('NRT_OBSERVATION', 'RDB_MODERN', 'SELECT * FROM nrt_observation WHERE last_chg_time :operator :timestamp OR refresh_datetime :operator :timestamp;', NULL, 'SELECT COUNT(*) FROM nrt_observation WHERE last_chg_time :operator :timestamp OR refresh_datetime :operator :timestamp;', 'WITH PaginatedResults AS (
+        SELECT *, ROW_NUMBER() OVER (ORDER BY (SELECT NULL)) AS RowNum
+        FROM nrt_observation
+        WHERE last_chg_time :operator :timestamp OR refresh_datetime :operator :timestamp
+    )
+    SELECT * FROM PaginatedResults
+    WHERE RowNum BETWEEN :startRow AND :endRow;')
+;
+
+
+INSERT INTO [RDB].[dbo].[data_sync_config]
```

Comment:
You don't need to specify the `[RDB]` prefix. 
This should be sufficient.
```INSERT INTO [dbo].[data_sync_config]```

In the deployment runtime, we will make a connection to the database and it should take care of it.